rustfmt.toml

```diff
@@ -1 +1,5 @@
 edition = "2021"
```

Comment:
According to the doc, the default strategy of resolving `edition` looks good, why would we override it?